libs/providers/ofrep/src/lib/ofrep-provider.ts

```diff
@@ -0,0 +1,80 @@
+import { EvaluationContext, JsonValue, Provider, ResolutionDetails, TypeMismatchError } from '@openfeature/server-sdk';
+import {
+  EvaluationFlagValue,
+  handleEvaluationError,
+  OFREPApi,
+  OFREPApiEvaluationResult,
+  OFREPProviderBaseOptions,
+  toRequestOptions,
+  toResolutionDetails,
+} from '@openfeature/ofrep-core';
+
+export type OFREPProviderOptions = OFREPProviderBaseOptions;
+
+export class OFREPProvider implements Provider {
+  private ofrepApi: OFREPApi;
+
+  readonly runsOn = 'server';
+  readonly metadata = {
+    name: OFREPProvider.name,
+  };
+
+  constructor(private options: OFREPProviderOptions) {
+    this.ofrepApi = new OFREPApi(options.baseUrl, options.fetchImplementation);
```

Comment:
Should we validate the base URL in the OFREPApi constructor and provide a friendly error if it's invalid?

Reply:
We ware doing it in all evaluations.